modules/light-clients/08-wasm/types/keys.go

```diff
@@ -12,11 +12,11 @@ const (
 	// StoreKey is the store key string for 08-wasm
 	StoreKey = ModuleName
 
-	KeyCodeIDPrefix = "codeId"
+	KeyCodeHashPrefix = "codeHash"
```

Comment:
Unfortunately this changes the store key prefix...

Reply:
This will change with https://github.com/cosmos/ibc-go/issues/4085 anyways tho right? we won't be storing the wasm code anymore

Reply:
Yes, but I guess we will have a `codeHashes` key where we store the list of all code hashes.